### ISSUE TITLE
Error when serving theme with minimal data

### ISSUE THREAD

**Issue by oakc123** [digitalcraftsman/hugo-material-docs#7]
I trying out the material docs theme I have a single post and a single page each with identical content.  Each has 2 images placed in the images directory of the theme.  After executing the server I get the following error:

ERROR: 2016/03/21 reflect: call of reflect.Value.Type on zero Value in theme/partials/drawer.html
ERROR: 2016/03/21 reflect: call of reflect.Value.Type on zero Value in theme/partials/footer_js.html
ERROR: 2016/03/21 reflect: call of reflect.Value.Type on zero Value in theme/partials/drawer.html
ERROR: 2016/03/21 reflect: call of reflect.Value.Type on zero Value in theme/partials/footer_js.html

**Comment by digitalcraftsman** [comment]
Thanks for moving to Github.

Sometimes the error messages of Hugo like above aren't really descriptive. Would it be possible to create a temporary repo of your Hugo website that I could clone. This makes it easier to locate the error since I don't need to guess your current setup.

**Comment by oakc123** [comment]
I can certainly try.  I have to say that I am not very versed in using git or Github.  So this might take me a few.

**Comment by digitalcraftsman** [comment]
Alternatively, zip your whole page and send it me via email at digitalcraftsman@protonmail.ch.

**Comment by oakc123** [comment]
Email sent ... but it returned.   <digitalcraftsman@protonmail.ch>: Recipient address rejected: this address does not exist

**Comment by digitalcraftsman** [comment]
I'm sorry, it's digitalcraftsman@protonmail***.com***

**Comment by digitalcraftsman** [comment]
**TLDR**: The mistake is trivial but as a new user you might not aware of it. Some themes require a more complex config file that sets default values. Therefore, many themes have a [`exampleSite`](https://github.com/digitalcraftsman/hugo-material-docs/tree/master/exampleSite) folder that contains such a config file. Copy the `config.toml` from there and replace the the original one in the root directory with it.

Since the default config file generated by Hugo doesn't contain these values the theme throws an error because the values are required. 

Initially, this folder was introduced to allow [theme demos](https://themes.gohugo.io) to have their own config file, content files etc. This way we such files can be easily distributed with the theme too. It's mostly a good idea to read the `README.md` file. This mostly the first place where you find further instructions.
